docs/source/getting-started/docker-machine-drivers.adoc

```diff
@@ -108,6 +108,26 @@ For more information, see the GitHub documentation of the https://github.com/zch
 
 To manually install the xhyve driver, run the following commands:
 
+. Download the xhyve binary using:
++
+----
+$ sudo curl -L  https://github.com/zchee/docker-machine-driver-xhyve/releases/download/v0.3.1/docker-machine-driver-xhyve -o /usr/local/bin/docker-machine-driver-xhyve
+----
+
+. Docker-machine-driver-xhyve needs root access and uid, set your permissions using:
++
+----
+$ sudo chown root:wheel /usr/local/bin/docker-machine-driver-xhyve
+----
+
+. Execute the xhyve binary as follows:
```

Comment:
It should be "Set owner User ID (SUID) for docker-machine-driver-xhyve binary as follows:"